src/scroll-timeline-css-parser.js

```diff
@@ -84,9 +84,17 @@ export class StyleParser {
     }
 
     // If this sheet has no srcURL (like from a <style> tag), we are done.
-    // TODO: Otherwise, we have to find `url()` functions and resolve
-    // relative and path-absolute URLs to absolute URLs.
-    return p.sheetSrc;
+    // Otherwise, we have to find `url()` functions and resolve
+    // relative URLs to absolute URLs.
+    if (!srcUrl) {
+      return p.sheetSrc;
+    }
+
+    const srcUrlDir = srcUrl.lastIndexOf('/') > location.origin.length
+        ? srcUrl.substring(0, srcUrl.lastIndexOf('/'))
+        : location.origin;
+
+    return p.sheetSrc.replace(/url\((["'])?(?:\.?\/|(?!https?:\/\/|(?:data|blob):))/gm, `url($1${srcUrlDir}/`);
```

Comment:
Correct me if I'm wrong, but I think there are a couple issues with this replacement:
1. The optional dot `\.?` seems like it will treat `url('./foo.png')` and `url('/foo.png')` the same, but shouldn't the latter be relative to the root of the associated url?
2. It seems like it replaces quoted data URI's, e.g. try it with `url('data:text/css,bar')`, it inserts the `srcUrlDir` before the quote.

Reply:
Fixed in 24ba542